Cargo.lock

```diff
@@ -93,6 +94,26 @@ dependencies = [
 ]
 
 [[package]]
+name = "errno"
```

Comment:
Regenerating it does not remove the `errno` entry - it just updates dependencies to the latest patch-levels. I am not sure if this is what you intended.